modules/util/source-formatter/src/main/resources/dependencies/replacements.json

```diff
@@ -3352,6 +3352,22 @@
 		"issueKey": "LPD-36648",
 		"to": "getSearchContainer"
 	},
+	{
+		"classNames": [
+			"Address",
+			"EmailAddress",
+			"OrgLabor",
+			"Phone",
+			"Website"
+		],
+		"from": "regex:(?<variableName>\\w+)\\.((?<prefixMethodName>(get|set))(TypeId))",
```

Comment:
@rubenssoares-br we could simplify it to this:
`(?<variableName>\\w+)\\.(?<prefixMethodName>get|set)TypeId` ?

It has more parenthesis than necessary. What do you think?